src/Voron/Data/Tables/Table.cs

```diff
@@ -903,6 +906,9 @@ public IEnumerable<SeekResult> SeekForwardFrom(TableSchema.SchemaIndexDef index,
         public TableValueHolder SeekOneForwardFromPrefix(TableSchema.SchemaIndexDef index, Slice value)
         {
             var tree = GetTree(index);
+            if (tree == null)
```

Comment:
should we check that number of entries in 0 here?

Reply:
it's a perf thingy but yeah

Reply:
I don't think we should do that, it is an already handled scenario

Reply:
Small optimization to avoid creating iterator, setting prefix, seeking, etc. I do not see a good reason not to have this?

Reply:
I can't think of any scenario where this would be meaningful performance optimization.
Null / empty trees are rare, but you've added a new conditional for every call

Reply:
Removed this from 6 locations: https://github.com/ravendb/ravendb/pull/9188